dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -375,12 +375,14 @@ export const LiveLayout = ({ CAPI, NAV, format, palette }: Props) => {
 
 	const showOnwardsLower = seriesTag && CAPI.hasStoryPackage;
 
-	const showComments = CAPI.isCommentable;
-
-	const showMatchTabs = CAPI.matchUrl;
-
 	const age = getAgeWarning(CAPI.tags, CAPI.webPublicationDateDeprecated);
 
+	// Set a default pagination if it is missing from CAPI
+	const pagination: Pagination | false = CAPI.pagination ?? {
+		currentPage: 1,
+		totalPages: 1,
+	};
```

Comment:
we don't seem to use the `T | false` anywhere else in the repo. Also not really sure how this can ever be false?  I think the type is just `Pagination`?

Reply:
Nothing to see here… 😮‍💨